ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx

```diff
@@ -12,16 +12,28 @@
 // limitations under the License.
 
 import { produce } from 'immer';
-import { Stack, Box, Typography } from '@mui/material';
+import { Button, IconButton, Stack, Typography } from '@mui/material';
+import AddIcon from 'mdi-material-ui/Plus';
+import DeleteIcon from 'mdi-material-ui/DeleteOutline';
 import { TimeSeriesQueryDefinition } from '@perses-dev/core';
-import { OptionsEditorProps, TimeSeriesQueryEditor } from '@perses-dev/plugin-system';
+import { OptionsEditorProps, TimeSeriesQueryEditor, usePlugin } from '@perses-dev/plugin-system';
+
 import { TimeSeriesChartOptions } from './time-series-chart-model';
 
+const DEFAULT_QUERY_PLUGIN_TYPE = 'TimeSeriesQuery';
+const DEFAULT_QUERY_PLUGIN_KIND = 'PrometheusTimeSeriesQuery';
+
 export type TimeSeriesChartOptionsEditorProps = OptionsEditorProps<TimeSeriesChartOptions>;
 
 export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditorProps) {
```

Comment:
Oh man, this file is about to get huge after my [legend options PR](https://github.com/perses/perses/pull/660/files#diff-bfad4a5ad2e72f83d687801cdb2b390252da88da4dcd59ddf3b027c0c359c514) gets merged in 😂 (I didn't add the 'Query' and 'Visualization' tabs yet so it'll get better after that)